code/modules/research/designs/boards/machine_science.dm

```diff
@@ -103,22 +103,22 @@
 	id = "accelerator"
 	build_path = /obj/item/weapon/circuitboard/anom/accelerator
 
-/datum/design/anom/accelerator
```

Comment:
Oh this is just the worst, I hope pjb doesn't see this or he'll have an aneurysm.